ha/setup/pre_disruptive_upgrade.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import os
+from shutil import copytree, rmtree
+import tarfile
+from time import gmtime, strftime
+from cortx.utils.log import Log
+from ha.execute import SimpleCommand
+import subprocess
+import shlex
+from typing import Any
+from xml.etree import ElementTree
+from xml.etree.ElementTree import Element
+
+# TODO: add docstrings ones approach is settled
+
+# FIXME: Discuss where to take this constant. Confstore of HA config file?
+BACKUP_SOURCE_DIR="/etc/cortx/ha"
+BACKUP_DEST_DIR="/opt/seagate/cortx/ha_conf_backup"
+# FIXME: Why do we need Hare directory in original script?
+HARE_DIR="/var/lib/hare"
```

Comment:
In original script consul dump was stored in hare directory. I don't think that it is good idea to use directories that belong to another component. Better to store it somewhere else. @ajaysrivas what do you think?

Reply:
Let's use backup directory for HA.
BACKUP_DEST_DIR="/opt/seagate/cortx/ha_backup"
BACKUP_DEST_DIR_CONF="/opt/seagate/cortx/ha_backup/conf"
BACKUP_DEST_DIR_CONSUL="/opt/seagate/cortx/ha_backup/Consul"

Reply:
Added these variables to const.py and used it.